docker/build-tools/Dockerfile

```diff
@@ -27,9 +27,9 @@
 
 FROM golang:1.13.4 as binary_tools_context
 
-# Istio tools branch that we use for this image
-ARG ISTIO_TOOLS_BRANCH
-ENV ISTIO_TOOLS_BRANCH=${ISTIO_TOOLS_BRANCH}
+# Istio tools SHA that we use for this image
+ARG ISTIO_TOOLS_SHA
+ENV ISTIO_TOOLS_SHA=${ISTIO_TOOLS_SHA}
```

Comment:
Why do we need env if we have arg? I realize it was this way before though

Reply:
That was my mental way how to make `ARG` work, clearly I was wrong. I will make the change.